src/Illuminate/Pagination/LengthAwarePaginator.php

```diff
@@ -50,7 +49,7 @@ public function __construct($items, $total, $perPage, $currentPage = null, array
         $this->lastPage = max((int) ceil($total / $perPage), 1);
         $this->path = $this->path !== '/' ? rtrim($this->path, '/') : $this->path;
         $this->currentPage = $this->setCurrentPage($currentPage, $this->pageName);
-        $this->items = $items instanceof Collection ? $items : Collection::make($items);
+        $this->items = collect($items);
```

Comment:
This check is anyway done in `\Illuminate\Support\Traits\EnumeratesValues::getArrayableItems`